src/C++/Session.cpp

```diff
@@ -366,7 +388,7 @@ void Session::nextResendRequest( const Message& resendRequest, const UtcTimeStam
   {
     endSeqNo = EndSeqNo(endSeqNo + 1);
     int next = m_state.getNextSenderMsgSeqNum();
-    if( endSeqNo > next )
+    if( endSeqNo >= next )
```

Comment:
Unreachable for current testing when does not include greater than or equal to and does not do any harm to set the value to eachother should they equal oneanother (as if they are equal the value change will essentially do nothing).  Did not remove check due to the chance that something above this execution changes where the check will be necessary.  Needed for 100% coverage